DQM/DTMonitorModule/python/dtDQMOfflineSources_HI_cff.py

```diff
@@ -50,3 +50,13 @@
                          dtEfficiencyMonitor +
                          dtTriggerEfficiencyMonitor +
                          dqmInfoDT)
+
+import EventFilter.DTRawToDigi.dturosunpacker_cfi
+_dturosunpacker = EventFilter.DTRawToDigi.dturosunpacker_cfi.dturosunpacker.clone()
+from Configuration.Eras.Modifier_run2_DT_2018_cff import run2_DT_2018
```

Comment:
era based modifications must be included in the python file that defines the module (and not propagated into the many other configurations that import it)

Reply:
I agree with @perrotta 's comment - this is the wrong spot for the change

Reply:
still a pending item (to be sure its not forgotten) - it appears in several spots below

Reply:
@davidlange6 I am confused: dtDataIntegrityUnpacker is defined in this python config file, hence teh replacement should be here, right?

Reply:
DTDataIntegrityTask is defined in DQM/DTMonitorModule/python/dtDataIntegrityTask_cfi.py

Reply:
DTDataIntegrityTask was defined here instead than in DQM/DTMonitorModule/python/dtDataIntegrityTask_cfi.py: now fixed